### DISCUSSION TITLE
a strange bug with creating channels on the server

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1626] [Ideas]
### Description

so you know that while creating a channel, there is a chek box called (join channel on exit chek box) this option allows you to join the channel that your creating as soon as you press ok button and create the channel: when we creating a channel, we changing all settings as how as we want, and then creating the channel as should be expect: but when this (join channel on exit) option is chekked and we finished creating the channel, the max users and Disk quota is not changed: max users and Disk quota is going back to default and not what as we set and we wanted to be: for example, i'm creating a channel, and i set my max users to 5, and my Disk quota to 1000000 and the (join channel on exit) option is also chekked. when i press ok button to complete creation of my channel, then i go to update channel menu to vizit, the max users is 1000, and Disk quota is 8000000: a fue notes: 1: this bug is not happening if (join channel on exit) option is not chekked, and everything works as expected. 2: as i know, this bug is only happening with max users and Disk quota but the others are fine and works as expected. stil if you  want, you can test it.: server version: 5.11

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

if the (join channel on exit) option is enabled, the max users and the Disk quota shouldn't break and should change and work whel.

### Actual behavior

when the (join channel on exit) option is enabled, the max users and Disk quota go's back to default and doesn't save.

### Steps to reproduce problem

creat a channel, edit your channel settings, and enable (join channel on exit) option. now if you press ok button and creat your channel, in the update channel section, you'll see that max users and Disk quota is not changed to what you wanted them to be and they are back to default: note: this bug doesn't exist if (join channel on exit) option is disabled

**Comment by bear101** [comment]
You can only change "max users" and "disk quota" as administrator.

**Comment by amirmahdifard** [comment]
i did it as administrator and this thing stil happened to me. btw sorry that i forgot to provide that i'm doing this with administrator account. but now i said.

**Comment by amirmahdifard** [comment]
@bear101 before i saw that you was active and pushed lots of commit, but these days yourvnot active slso much, y?

**Comment by bear101** [comment]
I've added this commit 8174e517604c3033187666ecb36b479a668a0994 where max users in channel is also inherited from server properties. Before it was only disk quota that was inherited.

When joining a channel max-users and disk quota are ignored. Both as administrator and default user.

**Comment by amirmahdifard** [comment]
@bear101 l99k like that this problem isn't fixed in 5.12, but you say that it is: can you tell me y?
